ui/next.config.mjs

```diff
@@ -6,11 +7,55 @@ const nextConfig = {
     ignoreDuringBuilds: true,
   },
   reactStrictMode: true,
-  // https://github.com/chungwu/combat-lander/commit/a22a0d2b302b623696b3e95d85566d3a7f1f6e3b
-  webpack: (config, { isServer }) => ({
-    ...config,
-    experiments: { asyncWebAssembly: true, layers: true },
-  }),
+  webpack: (config, options) => {
+    if (!options.isServer) {
+      config.resolve.fallback.fs = false
+      config.resolve.fallback.readline = false
+    }
+    config.experiments = { asyncWebAssembly: true, syncWebAssembly: true, layers: true }
+
+    config.plugins.push(
+      new CircularDependencyPlugin({
```

Comment:
do we really need this?
Have you tried without?

I believe I had fixed all circular deps issues in 
https://github.com/anonklub/anonklub/pull/467
(I had used this tool to detect them https://github.com/pahen/madge)

Reply:
We don't need it anymore, I just wanted to check any circular dependencies. So that is removed now.